open_spiel/python/pybind11/pyspiel.cc

```diff
@@ -271,7 +271,12 @@ PYBIND11_MODULE(pyspiel, m) {
 
   py::class_<Game, std::shared_ptr<Game>> game(m, "Game");
   game.def("num_distinct_actions", &Game::NumDistinctActions)
-      .def("new_initial_state", &Game::NewInitialState)
+      .def("new_initial_state",
```

Comment:
Can you use py::overload_cast instead?

https://pybind11.readthedocs.io/en/stable/classes.html#overloaded-methods

Reply:
Oh sure!